stablehlo/dialect/StablehloOps.cpp

```diff
@@ -71,7 +71,7 @@ limitations under the License.
 #include "mlir/Support/LogicalResult.h"
 #include "mlir/Transforms/InliningUtils.h"
 #include "stablehlo/dialect/StablehloBytecode.h"
-#include "stablehlo/dialect/StablehloOps.h.inc"
```

Comment:
Where did this include go?

Reply:
+1

Reply:
Good catch! Added back. Wondering it still worked even deleting it by mistake :-)